test/e2e/network_connect_disconnect_test.go

```diff
@@ -99,9 +99,13 @@ var _ = Describe("Podman network connect and disconnect", func() {
 	})
 
 	It("bad network name in connect should result in error", func() {
-		dis := podmanTest.Podman([]string{"network", "connect", "foobar", "test"})
+		session := podmanTest.Podman([]string{"create", "--name", "testContainer", "--network", "bridge", ALPINE})
+		session.WaitWithDefaultTimeout()
+		Expect(session).Should(ExitCleanly())
```

Comment:
This test was completely broken. PLEASE REVIEW CAREFULLY! I don't know if this is the test intention.

Reply:
looks good